integrations/opennms-vmware/src/main/resources/META-INF/opennms/component-dao.xml

```diff
@@ -33,7 +33,6 @@
     </bean>
 
     <bean id="vmwareConfigDao" class="org.opennms.netmgt.dao.vmware.jaxb.VmwareConfigDaoJaxb">
-        <property name="configResource" ref="vmwareConfigResourceLocation"/>
     </bean>
```

Comment:
using / to close if there is no property

Reply:
I mean this way e.g. 
`<bean id="vmwareConfigDao" class="org.opennms.netmgt.dao.vmware.jaxb.VmwareConfigDaoJaxb"/>
`